changelog/1.21.0.md

```diff
@@ -17,4 +17,5 @@ There are no bug fixes in 1.21.0.
 
 ### Security updates 🔐
 
-There are no security updates in 1.21.0.
+- ssh: New keys are generated using the [ED25519 digital signature algorithm](https://datatracker.ietf.org/doc/html/rfc8709)
```

Comment:
Didn't this also cause some sort of breakage with existing ECDSA keys? That's what I noticed in our dogfood environment, though I'm unsure how to replicate

Reply:
I wasn't able to replicate the issue you faced, but it sounds like you may have regenerated the SSH key while the workspace was still running, which would result in the new key being updated on GitHub and in `coderd` while the workspace still contained the old key. Rebuilding the workspace would bring everything back in sync again, but this would happen regardless of the key algorithm used.

Reply:
Makes sense, thanks for double-checking!